Changelog.md

```diff
@@ -107,6 +107,7 @@ Compiler Features:
 Bugfixes:
  * Build System: Support versions of CVC4 linked against CLN instead of GMP. In case of compilation issues due to the experimental SMT solver support, the solvers can be disabled when configuring the project with CMake using ``-DUSE_CVC4=OFF`` or ``-DUSE_Z3=OFF``.
  * Tests: Fix chain parameters to make ipc tests work with newer versions of cpp-ethereum.
+ * Tests: Do not use negative denominators for ``boost::rational`` in the test. This breaks for ``boost >= 1.68`` and negative denominators never actually occur.
```

Comment:
Actually I think this clutters the changelog unnecessarily.

Reply:
Yeah ok - I actually considered not adding an entry, but then added one regardless... I'll remove it.